### ISSUE TITLE
Open images dropped onto the sideberry tab list the way Firefox opens images

### ISSUE THREAD

**Issue by Ardemus** [mbnuqw/sidebery#1640]
### TLDR
I'd like sidebery to open all of the images dropped onto it, and open them in a way where Firefox image extensions work normally.

If I had to choose between this feature and a fix for issue #1639, I'd choose this feature, but I'd be happy with either.

### Details
I normally use Firefox extensions on local images by dropping one or more onto the blank area of a new tab. Firefox opens the first image in that tab, and additional images in other new tabs. It displays a file url like `file:///home/nick/Downloads/Screenshot%202024-05-09%201950.jpg` in the address bar and the tab icon is a generic image icon.

Sideberry currently prevents opening images that way (see issue #1639).

As a work around I tried to drop the images onto the Sidebery tab list. Sidebery does open one of the images (but only one, ignoring the rest). The address bar shows a blob url like `blob:moz-extension://e871a704-cbd3-471e-af93-45487adbf1e0/1a824d8f-fd3c-4898-84e0-f170450bbe01` and the tab icon is a Firefox logo. Image extensions do not work with the image. If an extension is normally activated from the right click menu, it does not appear in that menu. If it's normally activated when hovering the mouse over an image, or by clicking on an address bar button, it does not activate.

**Comment by Kryptortio** [comment]
Same thing happens for html and text files. But when I try it it just says "Unavailable URL"

```html
<!DOCTYPE html>
<html lang="en" color-scheme="system-color"><head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <style>
    :root { color-scheme: light dark; background-color: light-dark(#eee, #333); }
  </style>
  <link rel="stylesheet" type="text/css" href="../themes/proton/url.css" id="theme_link">
  <title>blob:moz-extension://5028bce0-b3a5-4a8d-a5bf-b770d2ee4597/9a222b82-9e0c-4a75-ac23-7339efabf125</title>
</head>
<body id="root" data-theme="proton" data-frame-color-scheme="dark" data-toolbar-color-scheme="dark" style="--s-frame-bg: rgb(28, 27, 34); --s-frame-fg: #fbfbfe; --s-toolbar-bg: rgb(43,42,51); --s-toolbar-fg: rgb(251,251,254); --s-act-el-bg: rgba(106,106,120,0.7); --s-act-el-fg: rgb(255,255,255); --s-popup-bg: rgb(66,65,77); --s-popup-fg: rgb(251,251,254); --s-popup-border: rgb(82,82,94);">
  <div id="moz_dialog_color_scheme_probe" style="background-color: -moz-dialog; color: -moz-dialogtext; position: absolute; width: 0; height: 0;">
  </div>

  <h2 id="title">Unavailable URL</h2>
  <p id="title_note">Sidebery is unable to open this link due to security-related restrictions of Firefox.</p>

  <main>
    
    
    <p id="target_url_label" class="label">Original URL</p>
    <p id="target_url" class="value">blob:moz-extension://5028bce0-b3a5-4a8d-a5bf-b770d2ee4597/9a222b82-9e0c-4a75-ac23-7339efabf125</p>
    <div id="copy_btn" class="btn" tabindex="0">Copy URL</div>
  </main>

</body></html>
```

If I copy the url (including blob:) open a new tab and try to navigate to it the file does get displayed. Using `file:///` rather than `blob:moz-extension://` would be better though if that's possible.
